bitnami/vault/templates/server/statefulset.yaml

```
@@ -274,5 +276,32 @@ spec:
         {{- if .Values.server.persistence.selector }}
         selector: {{- include "common.tplvalues.render" (dict "value" .Values.server.persistence.selector "context" $) | nindent 10 }}
         {{- end }}
+    - metadata:
+        name: plugin
+        annotations:
+          {{- if .Values.server.persistence.annotations }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.server.persistence.annotations "context" $) | nindent 10 }}
+          {{- end }}
+          {{- if .Values.commonAnnotations }}
+          {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 10 }}
+          {{- end }}
+        {{- if .Values.commonLabels }}
+        labels: {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 10 }}
+        {{- end }}
+      spec:
+        {{- if .Values.server.persistence.dataSource }}
+        dataSource: {{- include "common.tplvalues.render" (dict "value" .Values.server.persistence.dataSource "context" $) | nindent 4 }}
+        {{- end }}
+        accessModes:
+        {{- range .Values.server.persistence.accessModes }}
+          - {{ . | quote }}
+        {{- end }}
+        resources:
+          requests:
+            storage: {{ .Values.server.persistence.size | quote }}
+        {{- include "common.storage.class" (dict "persistence" .Values.server.persistence "global" .Values.global) | nindent 8 }}
+        {{- if .Values.server.persistence.selector }}
+        selector: {{- include "common.tplvalues.render" (dict "value" .Values.server.persistence.selector "context" $) | nindent 10 }}
+        {{- end }}        
```

What do you think about replacing this piece of code with something like this:
```suggestion
        {{- if .Values.server.persistence.extraVolumeClaimTemplates }}
        {{- include "common.tplvalues.render" (dict "value" .Values.server.persistence.extraVolumeClaimTemplates "context" $) | nindent 8 }}
        {{- end }}
```

---

okay, This will also introduce a new param extraVolumeClaimTemplates in values.yaml right?